esp/src/eclwatch/PackageMapValidateWidget.js

```diff
@@ -211,7 +211,7 @@ define([
                 }
             }).then(function (response) {
                 if (lang.exists("ValidatePackageResponse", response)) {
-                    var responseText = context.validateResponseToText(response.ValidatePackageResponse);
+                    var responseText = context.validateResponseToText(response.ValidatePackageResponse.Results.Result[0]);
```

Comment:
Is it possible that there might be no entries?

Reply:
I hadn't seen any examples where that was the case on clusters that I have access to, but I replaced that with an optional chaining operator